### ISSUE TITLE
Feature request - bypass bubble creation for PSP / PSX

### ISSUE THREAD

**Issue by jimbob4000** [VitaHEX-Games/HexFlow-Launcher#8]
If possible it would be great to bypass the need for bubbles for non vita games. 

The vita has a bubble limit which may prevent the full collection being imported. 

Maybe explore the code in Vita Launcher for a possible workaround. 
(Image matching may be an issue, iso games; may need to scan the iso sfo file to get the game ID)

Thanks for the awesome app.

**Comment by BeemerDream** [comment]
Yes, so we can see our legaly downloaded games (PS and PSP) in this app like the PS Vita games and start them normal, not via Adrenaline.

Great app by the way.

**Comment by jimbob4000** [comment]
Another route would could be to discuss with the author of the adrenaline bubble manager:
https://github.com/ONElua/AdrenalineBubbleManager

As a workaround for the moment, I am removing the bubbles from the app.db "tbl_appinfo_icon".
They could implement a "hidden" bubble install feature which could install to the app folder but hide the icon?

Or perhaps author a new app to manage the visibility of bubbles.

**Comment by HitmanHighman** [comment]
JimBob, could you possibly give a step by step on your hidden bubble workaround? Sounds like a great idea for now. Do the hidden bubbles still count toward the 500 limit? Thanks

**Comment by mikesierra85** [comment]
> JimBob, could you possibly give a step by step on your hidden bubble workaround? Sounds like a great idea for now. Do the hidden bubbles still count toward the 500 limit? Thanks

It doesn't count towards the count as there won't be any visible bubbles on the interface, I have over 1000 Vita games that all show up in Vita Hex Launcher after I've hidden all the bubbles from app.db, it's done within a minute if you use a macro, in a nutshell, you need to change the app page position to say something like 100, as the Vita won't have 100 pages anyway, so that makes the bubbles out of the range and won't be visible.
